pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -91,7 +91,7 @@ func (s *Server) initConfigController(args *PilotArgs) error {
 		}
 		s.ConfigStores = append(s.ConfigStores, configController)
 		if features.EnableServiceApis {
```

Comment:
Any reason to not have this on by default ( at least in master) ?

Reply:
There was a good reason at the time, we would fail if the CRD wasn't present. That has since been fixed, so we can probably default on in the future